tensorboard/plugins/hparams/tf_hparams_sessions_pane/tf-hparams-sessions-pane.ts

```diff
@@ -169,4 +184,23 @@ class TfHparamsSessionsPane extends PolymerElement {
     type: Number,
   })
   _selectedTab: number = 0;
+  _tableTabClicked: () => void = () => {
+    this.logTabClick('Table');
+  };
+  _parrellelCoordsTabClicked: () => void = () => {
+    this.logTabClick('Parrellel Coords');
+  };
+  _scatterPlotMatrixTabClicked: () => void = () => {
+    this.logTabClick('Scatter Plot Matrix');
+  };
+
+  logTabClick: (tabName: string) => void = (tabName: string) => {
+    // @ts-ignore
+    this.ga('send', {
+      hitType: 'event',
+      eventCategory: 'HParam',
```

Comment:
Nit: maybe "HParams" since technically the name of the plugin is the plural form:
https://github.com/tensorflow/tensorboard/blob/6e7fa14b0b32e6c4ef4e7fb9d77bd9739cd69e50/tensorboard/plugins/hparams/metadata.py#L25